02LessonTwo/homework/index.html

```diff
@@ -1,10 +1,29 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
+  <title>Nicole D'Augereaux</title>
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+      <div><center><h1>Nicole D'Augereaux</h1></center></div>
+        <center><img src="https://img1.etsystatic.com/104/0/76058609/iusa_400x400.36289367_sv7h.jpg"/></center>
```

Comment:
make sure to give returns and tabs when things are nested. lines 8, 9, 12, and 21 have this issue. Great job commenting!